Index: sla/2.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>//\r\n// Created by 26538 on 2021/1/22.\r\n//\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/sla/2.cpp b/sla/2.cpp
--- a/sla/2.cpp	(revision df9d913b6cb681d8aea902e26414ec654e8d0f2e)
+++ b/sla/2.cpp	(date 1611308559995)
@@ -1,4 +1,10 @@
 //
 // Created by 26538 on 2021/1/22.
 //
+#include<iostream>
+using namespace std;
+int main(){
+
+}
+
 
Index: sla/3.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/sla/3.cpp b/sla/3.cpp
new file mode 100644
--- /dev/null	(date 1611308560001)
+++ b/sla/3.cpp	(date 1611308560001)
@@ -0,0 +1,10 @@
+//
+// Created by 26538 on 2021/1/22.
+//
+#include<iostream>
+using namespace std;
+int main(){
+
+}
+
+
